src/views/Instance/data.js

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import axios from 'axios';
+import { Auth } from '@cosmotech/core';
+import {
+  DEFAULT_EDGE_STYLE,
+  DEFAULT_NODE_STYLE,
+  DEFAULT_SELECTED_EDGE_STYLE,
+  DEFAULT_SELECTED_NODE_STYLE,
+} from './styleCytoViz';
+import { ORGANIZATION_ID, WORKSPACE_ID } from '../../config/AppInstance';
+import instanceViewData from '../../config/InstanceVisualization.js';
+
+const IS_INSTANCE_VIEW_FUNCTION_CONFIG_VALID = !(instanceViewData.dataSource == null);
+export { IS_INSTANCE_VIEW_FUNCTION_CONFIG_VALID };
+
+const _formatLabelWithNewlines = (label) => label?.replace(/[_|\s]/g, '\n') || '';
+
+const _forgeCytoscapeNodeData = (node, classes, nodesGroupMetadata = {}, nodesParentsDict = {}) => {
```

Comment:
IMO, it makes the code simpler to write because we don't have to check whether these parameters are null or not:
```
  const parent = nodesParentsDict[node.id];
  const { style, ...otherMetadata } = nodesGroupMetadata;
```